packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityStakeStep/SignStakeTransactions/SignStakeTransactions.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { CWDivider } from 'views/components/component_kit/cw_divider';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import ActionSteps from '../../../components/ActionSteps';
+import CreateCommunityHint from '../../../components/CreateCommunityHint';
+
+import './SignStakeTransactions.scss';
+
+interface SignStakeTransactionsProps {
+  onOptOutEnablingStake: () => void;
+}
+const SignStakeTransactions = ({
+  onOptOutEnablingStake,
+}: SignStakeTransactionsProps) => {
+  const renderHint = (className: string) => (
+    <CreateCommunityHint
+      className={className}
+      title="What is a namespace?"
+      hint="The namespace is an address that represents your community  on-chain.
+          You can purchase additional community namespaces from the admin panel."
+    />
+  );
```

Comment:
i saw this used above, maybe we can make it a common component/util?

Reply:
created new component